Astronomer tap optional if cli install fails (#2949)
astronomer/docs: astro/cli/install-cli.md

Follow this procedure when Homebrew fails to install the latest Astro CLI version or the error `No formulae or casks found for astro@<major.minor.patch-version>` appears. To troubleshoot other Homebrew issues, see [Common Issues](https://docs.brew.sh/Common-Issues) in the Homebrew documentation.

1. If the install process is still not working, run the following command to update Homebrew and all package definitions (formulae):

    ```sh
    brew update
    ```
    
2. Re-run the installation again:
   
   ```sh
   brew install astro
   ```

3. If updating Homebrew doesn't work and this is the first time you're installing the CLI, check to see if `astronomer/tap` is in your [Homebrew Tap](https://docs.brew.sh/Taps) list: 

  ```sh
  brew tap astronomer/tap
  ```

> 4. Install the Astronomer CLI. You can choose to specify the major and minor version, if you want a particular version by adding the `astro@<major.minor.patch-version>` flag to the following command:

Reviewer: ```suggestion
4. Install the Astronomer CLI. You can choose to specify the major and minor version, if you want a particular version, you can add the `astro@<major.minor.patch-version>` flag to the `brew install` command:
```

> 4. Install the Astronomer CLI. You can choose to specify the major and minor version, if you want a particular version, you can add the `astro@<major.minor.patch-version>` flag to the `brew install` command:

Reviewer: ```suggestion
4. Install the Astronomer CLI. You can choose to specify the major and minor version by adding the `astro@<major.minor.patch-version>` flag to the `brew install` command:
```

   ```sh

>    brew install astro

Reviewer: ```suggestion
    brew install astro@<major.minor.patch-version>
```
